tests/network/service_mesh/conftest.py

```diff
@@ -441,3 +435,10 @@ def vmi_http_server(vm_fedora_with_service_mesh_annotation):
         vm=vm_fedora_with_service_mesh_annotation,
         commands=[f'while true ; do  echo -e "HTTP/1.1 200 OK\n\n $(date)" | nc -l -p {SERVICE_MESH_PORT}  ; done &'],
     )
+
+
+@pytest.fixture(scope="module")
+def service_mesh_tests_namespace(namespace, admin_client):
+    # The namespace used for the ServiceMesh tests must be added the `istio-injection` label.
+    label_project(name=namespace.name, label={"istio-injection": "enabled"}, admin_client=admin_client)
```

Comment:
```suggestion
def service_mesh_tests_namespace():
    # The namespace used for the ServiceMesh tests must be added the `istio-injection` label.
    yield from create_ns(
        name="test-service-mesh-ns",
        labels={"istio-injection": "enabled"},
    )
```
Will that work?
Its more straight forward.

Reply:
It should work, but I prefer not to do it this way.
Why? Because I prefer being consistent with the common convention in cnv-tests, where each test module has its dedicated namespace which is created by the `namespace` fixture, and if necessary - apply any needed configuration (in this case - this additional label) on that namespace.
Does that make sense?